src/site/content/en/blog/richer-offline-w-periodic-background-sync/index.md

```diff
@@ -0,0 +1,458 @@
+content will refresh after launch, leading to content changing while the user is
+viewing it. Worse yet, the app may only show a content spinner.
+
+Without periodic background sync, web apps must use alternative methods to
+download data. A common example is using a push notification to wake a service
+worker. The user is interrupted by a message such as 'new data available'.
+Updating the data is essentially a side effect. You still have the option of
+using push notifications for truly important updates, such as significant
+breaking news.
+
+Periodic background sync is easily confused with background sync. Though they
+have similar names, their use cases are different. Among other things,
+background sync is most commonly used for resending data to a server when a
+previous request has failed.
+
+### Getting this right
+
+We are putting periodic background sync through a trial period so that you can
+help us make sure that we get it right. This section explains some of the design
+decisions we took to make this feature as helpful as possible.
+
+The first design decision we made is that a web app can only use periodic
+background sync after a person has installed it on their device, and has
+launched it as a distinct application. Periodic background sync is not available
+in the context of a regular tab in Chrome.
+
+Furthermore, since we don't want unused or seldom used web apps to gratuitously
+consume battery or data, we designed periodic background sync such that
+developers will have to earn it by providing value to their users. Concretely,
+we are using a site engagement score to determine if and how often periodic
+background syncs can happen for a given web app. In other words, a
+`periodicsync` event won't be fired at all unless the engagement score is
+greater than zero, and its value affects the frequency at which the
+`periodicsync` event fires. This ensures that the only apps syncing in the
+background are the ones you are actively using.
+
+Periodic background sync shares some similarities with existing APIs and
+practices on popular platforms. For instance, one-off background sync as well as
+push notifications allow a web app's logic to live a little longer (via its
+service worker) after a person has closed the page. On most platforms, it's
+common for people to have installed apps that periodically access the network in
+the background to provide a better user experience for critical updates,
+prefetching content, syncing data, etc. Similarly, periodic background sync also
+extends the lifetime of a web app's logic to run at regular periods for what
+might be a few minutes at a time.
+
+If the browser allowed this to occur frequently and without restrictions, it
+could result in so me privacy concerns. Here's how Chrome has addressed this
+risk for periodic background sync:
+
+* The background sync activity only occurs on a network that the device has
```

Comment:
Is this a cellular network like "ATT", or a Wi-Fi network like "GoogleGuest"?

Reply:
I don't know. It's not in any of the source materials. I can check with Beverloo, but this is a paragraph that sec/priv won't let us change.